charts/gpu-metrics-exporter/templates/daemonset.yaml

```diff
@@ -5,68 +5,69 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "gpu-metrics-exporter.labels" . | nindent 4 }}
+  {{- if .Values.dcgmExporter.enabled }}
   annotations:
-    {{- if .Values.dcgmExporter.enabled }}
     ignore-check.kube-linter.io/privileged-container: "This daemon set needs to run DCGM Exporter as privileged to access the GPU metrics."
     ignore-check.kube-linter.io/run-as-non-root: "This daemon set needs to run DCGM Exporter as root to access the GPU metrics."
-    ignore-check.kube-linter.io/privilege-escalation-container: "This daemon set needs escalate privileges for DCGM Exporter to access the GPU metrics."
-    ignore-check.kube-linter.io/no-read-only-root-fs: "This daemon set needs to run DCGM Exporter with read-only root filesystem to access the GPU utilization metrics."
-    {{- end }}
+    ignore-check.kube-linter.io/privilege-escalation-container: "This daemon set needs escalate privileges for DCGM Exporter."
+    ignore-check.kube-linter.io/no-read-only-root-fs: "This daemon set needs to run DCGM Exporter with read-only root filesystem."
+    ignore-check.kube-linter.io/drop-net-raw-capability: "This daemon set needs to run DCGM Exporter with SYS_ADMIN."
```

Comment:
`drop-net-raw-capability` refers to `NET_RAW`, how is that correlates with `SYS_ADMIN` ? 🤔

Reply:
it's the check that fails when we add the SYS_ADMIN capability :/

Reply:
> [drop-net-raw-capability](https://docs.kubelinter.io/#/generated/checks?id=drop-net-raw-capability)
> Description: Indicates when containers do not drop NET_RAW capability
> Remediation: NET_RAW makes it so that an application within the container is able to craft raw packets, use raw sockets, and bind to any address. Remove this capability in the containers under containers security contexts.

So, SYS_ADMIN let's you do NET_RAW stuff, and the check says we don't drop it. I'll test if it's going to be ok if I drop only NET_RAW but keep SYS_ADMIN